test/UnitTest/Microsoft.AspNetCore.OData.Test/Abstraction/RoutingConfigurationFactory.cs

```diff
@@ -74,12 +77,18 @@ public static IRouteBuilder Create()
 
             // Create a route build with a default path handler.
             IRouteBuilder routeBuilder = new RouteBuilder(appBuilder);
+
+#if NETCOREAPP3_0
+            routeBuilder.DefaultHandler = null;
+#else
```

Comment:
A few options to work around this:
- Write your own route handler for test purpose (mock)
- Use reflection to retrieve the type and get it from the service provider
- Call `UseMvc` and then use that route builder
- Come up with a different strategy for testing

I don't have the full context on what you're trying to do here.